packages/react-dom-bindings/src/client/ReactDOMComponent.js

```diff
@@ -273,6 +275,7 @@ function setProp(
   key: string,
   value: mixed,
   props: any,
+  prevValue: mixed,
```

Comment:
The `props` argument is only used by `dangerouslySetInnerHTML` to check that there's not also `children`. `prevValue` is only used by `style`. A possible alternative would be to special case these two in each loop instead.